clinguin_ui/ui_add_object.lp

```diff
@@ -0,0 +1,106 @@
+                        attr(addobject_dd,order,1).
+
+                        class_to_add(C) :- ooasp_leafclass(C).
+
+                            elem(dd_element(C),dropdown_menu_item,addobject_dd):-class_to_add(C).
+                            attr(dd_element(C),label,C):-class_to_add(C).
+                            when(dd_element(C),click,context,(class_name,C)):-class_to_add(C).
+                            when(dd_element(C),click,update,(addobject_dd,selected,C)):-class_to_add(C).
+
+
+                        elem(addobject_dd_amount,dropdown_menu,addobject_dd_c).
+                        attr(addobject_dd_amount,selected,"1").
+                        attr(addobject_dd_amount,class,("m-2";"p-2";"border";"border-1";"bg-primary";"bg-opacity-10")).
+                        attr(addobject_dd_amount,class,("disabled")):- _clinguin_browsing.
+                        attr(addobject_dd_amount,order,1).
+
+                            amount(1..10).
+                            elem(dd_element(C),dropdown_menu_item,addobject_dd_amount):-amount(C).
+                            attr(dd_element(C),label,C):-amount(C).
+                            attr(dd_element(C),order,10-C):-amount(C).
+                            when(dd_element(C),click,context,(amount,C)):-amount(C).
+                            when(dd_element(C),click,update,(addobject_dd_amount,selected,C)):-amount(C).
+
+                        elem(addobject_btn,button,addobject_dd_c).
+                        attr(addobject_btn,icon,"fa-plus").
+                        attr(addobject_btn,class,("btn-primary";"m-2";"border";"border-dark";"border-2")).
+                        attr(addobject_btn,class,("opacity-25")):- _clinguin_browsing.
+                        attr(addobject_btn,order,3).
+                        when(addobject_btn,click,call,add_object(_context_value(class_name,const,object),_context_value(amount,int,1))):- not _clinguin_browsing.
+                        when(addobject_btn,mouseenter,update,(add_warning,visibility,shown)):- _clinguin_browsing.
+
+                            elem(add_warning, message, w).
+                            attr(add_warning, message, "Adding objects is disabled while browsing").
+                            attr(add_warning, type, "warning").
+                            attr(add_warning, title, "Adding deactivated").
+                            attr(add_warning, visibility, hidden).
+
+            % Info container
+            elem(info_container,container,config_container).
+            attr(info_container,flex_direction,column).
+            attr(info_container,class,("col-sm";"flex-column")).
+            attr(info_container,order,1).
+
+                domain_size(S):- #count{1,ID:ooasp_domain(_,ID)} = S.
+                elem(domain_size_txt,label,info_container).
+                attr(domain_size_txt,label,@format("Domain Size: {}",DS)) :- domain_size(DS). %TODO fix sizing
```

Comment:
What exactly is to be done here?

Reply:
If I remember correctly this display issue was fixed already, it was just a styling issue.
I removed the comment.